webf/lib/src/dom/element.dart

```diff
@@ -556,6 +556,16 @@ abstract class Element extends Node with ElementBase, ElementEventMixin, Element
     renderBoxModel?.markAdjacentRenderParagraphNeedsLayout();
     // Ensure that the child is attached.
     ensureChildAttached();
+
+    // Reconfigure scrollable contents.
+    if (renderStyle.overflowX != CSSOverflowType.visible) {
+      updateRenderBoxModelWithOverflowX(_handleScroll);
+      updateOverflowRenderBox();
+    }
+    if (renderStyle.overflowY != CSSOverflowType.visible) {
+      updateRenderBoxModelWithOverflowY(_handleScroll);
+      updateOverflowRenderBox();
```

Comment:
if (renderStyle.overflowX != CSSOverflowType.visible || renderStyle.overflowY != CSSOverflowType.visible) {
  updateOverflowRenderBox();
}

Reply:
these will make the if-else twice.

Reply:
But updateOverflowRenderBox called it twice

Reply:
> 

It makes sense, does this method have an impact on performance if executed multiple times?